src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/ProjectInitializer.cs

```diff
@@ -34,7 +34,7 @@ private static string CombineFiles(string propsFile, string targetsFile)
         return tempTargetLocation;
     }
 
-    public static (Project, CapturingLogger, IDisposable) InitProject(Dictionary<string, string> bonusProps, Dictionary<string, ITaskItem[]>? bonusItems = null, [CallerMemberName]string projectName = "")
+    public static (Project, CapturingLogger, IDisposable) InitProject(Dictionary<string, string> bonusProps, Dictionary<string, ITaskItem[]>? bonusItems = null, [CallerMemberName] string projectName = "")
```

Comment:
This is helper code that lets us mock an MSBuild project in-memory. I found a small bug in test data setup while testing the actual change that I had to fix to let the test actually execute.